components/python/psycopg2/pkg5

```diff
@@ -1,16 +1,16 @@
 {
     "dependencies": [
         "SUNWcs",
-        "database/postgres-12/developer",
-        "database/postgres-12/library",
+        "database/postgres-14/developer",
+        "database/postgres-14/library",
         "runtime/python-37",
         "runtime/python-39",
         "shell/ksh93",
         "system/library"
     ],
     "fmris": [
-        "library/python/psycopg2-37",
-        "library/python/psycopg2-39",
+        "library/python/psycopg2",
+        "library/python/psycopg2",
         "library/python/psycopg2"
```

Comment:
@mtelka: Can you check why this happens?

Reply:
Probably a side effect of GENERATE_EXTRA_CMD=

Reply:
No, without that line the resulting pkg5 doesn't change.

Reply:
This is because you confused the build framework by asking for both: versioned python packages, and unversioned python packages.